Library/Formula/karn.rb

```diff
@@ -0,0 +1,38 @@
+require "language/go"
+
+class Karn < Formula
+  homepage "https://github.com/prydonius/karn"
+  url "https://github.com/prydonius/karn/archive/v0.0.2.tar.gz"
+  sha256 "eac77b82f7abd9c37e70d01240e8e69893b7497d3dd5ab7b0fb2df04ae1e6529"
+
+  depends_on "go" => :build
+
+  go_resource "github.com/codegangsta/cli" do
+    url "https://github.com/codegangsta/cli",
```

Comment:
If you change this to `https://github.com/codegangsta/cli.git` (and similarly for the next one) you won't need to specify `:using => :git`; it'll be able to figure it out.